roles/ec2-app/tasks/destroy_rds_instance.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: Destroy RDS instances for stack
+  rds:
+    command: delete
+    instance_name: "{{ rds_instance.db_name + '-' + opg_data.stack  }}"
+    snapshot: "{{ rds_instance.snapshot_name }}"
```

Comment:
No, on dev we don't want it to happen, or maybe we do, so if the name is not set, ie ansible `omit` it doesn't create the snapshot